test/test.js

```diff
@@ -3754,6 +3754,67 @@ var testFixture = {
             }
         },
 
+        'var o = {\n/** Desc */\nfoo: function(){}\n};': {
```

Comment:
No worries, you didn't. It seems that this issue is closed https://code.google.com/p/esprima/issues/detail?id=607 with AsExpected. Will I close this PR?